### ISSUE TITLE
Uncaught Error: Could not find code view table when enabling codecov decorations on github.com

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#5355]
Go to https://github.com/sourcegraph/sourcegraph/pull/5320/files and enable codecov background decorations. No decorations appear. Error in console (multiple times):

```
dom_functions.ts:239 Uncaught Error: Could not find code view table
    at isDomSplitDiff (dom_functions.ts:239)
    at getDiffCodeCellFromLineNumber (dom_functions.ts:77)
    at Object.getCodeElementFromLineNumber (dom_functions.ts:127)
    at applyDecorations (extensions.tsx:144)
    at SafeSubscriber.update [as _next] (code_intelligence.tsx:735)
    at SafeSubscriber.__tryOrUnsub (Subscriber.ts:266)
    at SafeSubscriber.next (Subscriber.ts:208)
    at Subscriber._next (Subscriber.ts:140)
    at Subscriber.next (Subscriber.ts:100)
    at FinallySubscriber.Subscriber._next (Subscriber.ts:140)
```

**Comment by felixfbecker** [comment]
This happens for code views with collapsed content. These should not be detected as code views

**Comment by felixfbecker** [comment]
Actually, they should be detected to get the open on Sourcegraph buttons, but not have decorations applied. It's tricky because we would want to apply them when they are expanded.

**Comment by felixfbecker** [comment]
The immediate error was fixed by #5358, fixing decorations is tracked in https://github.com/sourcegraph/sourcegraph/issues/5381
